test/spec/modules/openxBidAdapter_spec.js

```diff
@@ -615,18 +898,92 @@ describe('OpenxAdapter', () => {
 
     it('should register the pixel iframe from video ad response', () => {
       let syncs = spec.getUserSyncs(
-        { iframeEnabled: true },
-        [{ body: { pixels: syncUrl } }]
+        {iframeEnabled: true},
+        [{body: {pixels: syncUrl}}]
       );
-      expect(syncs).to.deep.equal([{ type: 'iframe', url: syncUrl }]);
+      expect(syncs).to.deep.equal([{type: 'iframe', url: syncUrl}]);
     });
 
     it('should register the default iframe if no pixels available', () => {
       let syncs = spec.getUserSyncs(
-        { iframeEnabled: true },
+        {iframeEnabled: true},
         []
       );
-      expect(syncs).to.deep.equal([{ type: 'iframe', url: '//u.openx.net/w/1.0/pd' }]);
+      expect(syncs).to.deep.equal([{type: 'iframe', url: '//u.openx.net/w/1.0/pd'}]);
     });
   });
+
+  /**
+   * Makes sure the override object does not introduce
+   * new fields against the contract
+   *
+   * This does a shallow check in order to make key checking simple
+   * with respect to what a helper handles.  For helpers that have
+   * nested fields, either check your design on maybe breaking it up
+   * to smaller, manageable pieces
+   *
+   * OR just call this on your nth level field if necessary.
+   *
+   * @param {Object} override Object with keys that overrides the default
+   * @param {Object} contract Original object contains the default fields
+   * @param {string} typeName Name of the type we're checking for error messages
+   * @throws {Error}
+   */
+  function overrideKeyCheck(override, contract, typeName) {
+    let invalidKey = Object.keys(override)
+      .find(key => !(key in contract));
+
+    if (invalidKey) {
+      throw new Error(`Mock Error: INVALID_KEY.  Key:${invalidKey} does not exist in type: ${typeName}`);
```

Comment:
This will stop execution of further unit tests. Wrap all calls to `overrideKeyCheck` in `try..catch`